README.rst

```diff
@@ -38,11 +38,12 @@ Just install docker from your package provider and run:
 Installing
 ==========
 
-We use Python 3.6, so you'll have to install it into your environment beforehand:
+If you don't have Python 3 installed, please install it. Please make
+sure that you're using ``python3.6`` or a later version:
 
 .. code-block:: shell
 
-   $ apt-get install python3.6
+   $ apt-get install python3
```

Comment:
You may need to add "sudo" here.
```suggestion
   $ sudo apt-get install python3
```

Reply:
Good catch, Carlos. Modern distributions recommend `apt` instead of `apt-get`, and `apt` already takes care of sudo'ing for the user, so I think we should standardize on that.